flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg

```diff
@@ -0,0 +1,63 @@
+[aircraft]
+icaotype = A20N
+parkingbrakestest = (L:A32NX_PARK_BRAKE_LEVER_POS,bool) (L:A32NX_MODEL_WHEELCHOKS_ENABLED,bool) ||
```

Comment:
Typo error: `A32NX_MODEL_WHEELCHOKS_ENABLED`
Regarding the model behaviour XML it should be `A32NX_MODEL_WHEELCHOCKS_ENABLED`

Reply:
Quality Assurance Trainee Report

Discord : LeDerpLegend#0109
Object of testing: #7695 
Tier of Testing : 1
Date : 16/01/2023

Testing Process:
Ensure simbrief loading still performs as before in EFB - GOOD

**Fueling**

- Test GSX properly plays the re-fueling animation when you begin refueling with the EFB - **OK**
- Test the refueling animation correctly stops when the EFB stops refueling the aircraft - **OK**
- Test the refueling animation is instant when you're using Instant refueling time - **OK**
- Ensure you can still refuel in-flight with GSX Fuel Sync enabled - **OK**

_Notes: Fuel button was NOT greyed out when requesting for a fuel truck, could fuel before truck arrived or hooked up._

**Payload**

- Set in the EFB how much passengers you want as-well as cargo, to notify GSX of these details. - **OK**
- Ensure passenger current in EFB, matches GSX tooltip - **OK**
- Confirm as GSX baggage tooltip percentage increase, current cargo weight increase in EFB - **FAIL**
- Ensure baggage weight decreases when deboarding - **FAIL**
- Ensure passenger amount decreases when deboarding - **FAIL**
- Ensure passenger/cargo amount in EFB cannot be modified after boarding is requested and only appears again after deboarding is completed - **FAIL**

_Notes: GSX PAX loaded super quickly in about 10 seconds, and I still have the ability to modify PAX and Payload between requesting boarding and fully deboarded._

Negatives:
- PAX and Payload connection with GSX not working as intended. 
- The ability to still modify fuel and payloads while GSX is preparing or between two stages should not be available. 
- Two catering vehicles still arrive at the plane, despite the front door not being able to open, however, the service still functions as normal.

Positives: 
- Only one baggage loader and cart spawned as intended, considering only one cargo bay door can be opened. (Restarting the Couatl service breaks this, and spawns two carts.)

Testing Result: NOT PASSED

Conclusions: Some GSX intergrations work as intended, but it seems there is not connection at all between the PAX/Payload EFB and GSX. GSX also seems to load PAX super fast. The blocking of fuel, payload, and passenger loading is not working as intended. I can still modify the fuel, passengers, and payload whenever, despite GSX's current position of vehicles and status of loading/unloading.

Reply:
resolved [3c420ef](https://github.com/flybywiresim/a32nx/pull/7695/commits/3c420ef6b7e05dbdafcd7d1d457722de51d7a24b)